packages/@react-spectrum/accordion/docs/Accordion.mdx

```diff
@@ -13,6 +13,7 @@ export default Layout;
 import docs from 'docs:@react-spectrum/accordion';
 import {HeaderInfo, PropTable, TypeLink, PageDescription} from '@react-spectrum/docs';
 import packageData from '@react-spectrum/accordion/package.json';
+import ChevronRight from '@spectrum-icons/workflow/ChevronRight';
```

Comment:
Is this needed?

Reply:
oh oops that's leftover from trying something else, i can follow-up on it